x-pack/plugins/index_management/public/application/components/component_templates/component_template_selector/component_templates_selector.scss

```
@@ -20,7 +20,7 @@
     &__header {
       background-color: $euiColorLightestShade;
       border-bottom: $euiBorderThin;
-      color: $euiColorInk;
+      color: $euiColorDarkestShade;
```

Would `$euiColorFullShade` be a better color variable choice (as it is the same color value as `$euiColorInk` in light mode, but will correctly change colors for dark mode)?

```suggestion
      color: $euiColorFullShade;
```